### ISSUE TITLE
PlayVitaOP is missing

### ISSUE THREAD

**Issue by AmayaMarie** [leycec/fsnrnue#31]
I can't get past Day 3. When I go to save Saber, I get an error message showing that Member PlayVitaOP is missing. I've updated the patches and installed and reinstalled Fate stay night and nothing changes. I hope you can figure it out bc googling hasn't helped me.
I hope you still check these.
Thank you and have a good day,
[PlayVitaOp log](https://github.com/leycec/fsnrnue/files/11379072/PlayVitaOp.log)

**Comment by leycec** [comment]
**Wow.** That's... absolutely horrible. I'm so sorry that's happened. What a bummer! Day 3's just when everything starts getting juicy, too. Thankfully, I've never hit this myself. Close call, guys. :sweat_smile: 

I acknowledge that doesn't help you very much. But I do have something that might. It's... a long-dead account on the Beast Lair, which reported **[the exact same issue two years ago](https://forums.nrvnqsr.com/showthread.php/8229-Fate-Stay-Night-Realta-Nua-Ultimate-Edition-%28thread-discontinued-see-OP-for-new-thread%29/page120):**

> another thing is that when i get to the "Save Saber or Run with Tohsaka" choice on the third day, and i save saber, when the text fades from white to black, it crashes with a message that says [Member "playVitaOp" does not exist] and links the file as [Saber Route Third Day-07.ks Line: 193] 

Unfortunately, no one actually replied to that specific issue with a working solution. I'm stumped.

@kaasknak: I summon thee to do deadly battle yet again! You got anything? No? You're equally stumped? Tremendously sorry, @AmayaMarie. It looks like we're useless. My face is exhaling. :face_exhaling: 

Would you mind posting your same exact question to the [official forum thread for the *Ultimate Edition Installer* at the Beast's Lair](https://forums.nrvnqsr.com/showthread.php/9101-Fate-Stay-Night-Realta-Nua-Ultimate-Edition-2022)? They're the experts and *should* be able to get you sorted. There's just one catch: whatever you do, **don't tell them that you're emulating on Linux under WINE.** If you do, they'll probably just politely ignore you. Of course, your logfile already gives everything away, but... I wouldn't go out of your way to advertise that you're on Linux.

**Gah!** Why does everything have to be so hard?

**Comment by kaasknak** [comment]
Hey @AmayaMarie 
Could you tell me whether you have installed and have certain settings activated in the vita menu?
If you have could you try turning them off?
You could also try downloading the patches from the alternative link in the README.md, quite a lot has changed in the windows installer on the beast's lair and we basically decided to not upgrade the version here since it breaks everything in glorious and explosive manners and neither I nor @leycec has had the time to burn on this (full time job and all that and in general changes in my life that don't allow me to spend as much time on projects such as these).

In short I propose the following:

- Turn off the patch settings for vita and see if that works
- Download the patches from the mirror
- Pray

Tell us if any of these worked or not so we know whether it worked or whether we need to dig deeper.

Good luck!

**Comment by ZedGreene** [comment]
did you find the solution to this problem, I am having the same trouble
